scapy/main.py

```diff
@@ -137,25 +138,38 @@ def save_session(fname=None, session=None, pickleProto=-1):
              log_interactive.error("[%s] (%s) can't be saved." % (k, type(to_be_saved[k])))
              del(to_be_saved[k])
 
-    try:
-        os.rename(fname, fname+".bak")
-    except OSError:
-        pass
+    
+    if os.path.exists(fname):
+        i=0
+        while True:
+            bak_f = fname+".bak"+("" if i==0 else str(i))
+            if not os.path.exists(bak_f):
+                os.rename(fname, bak_f)
+                break
+            i=i+1
+    
```

Comment:
I'm not sure if that's a good thing: what do you think ?
It allow to have endless saves of the session (instead of a single .bak, overwritten each time)... But is it relevent ?